### DISCUSSION TITLE
[FEATURE] 可以自定义lua_package_path

### DISCUSSION THREAD

**Discussion by 9a6c** [1Panel-dev/1Panel#5976] [Q&A]
### 1Panel 版本

1.10.13

### 请描述您的需求或者改进建议

尝试在OpenResty配置中增加lua_package_path配置，点击保存，出现报错。
![e1a47aeb386be8bd12f3182a7e49d61](https://github.com/user-attachments/assets/1bcba097-3b82-4acd-8310-df1cf09c3595)

希望可以自定义lua_package_path，目前waf配置中使用了lua_package_path，好像是不能多个地方进行配置。

### 附加信息

![image](https://github.com/user-attachments/assets/1e6f9a3d-f58d-424b-bc21-d777dc085821)

**Comment by zhengkunwang223** [comment]
![image](https://github.com/user-attachments/assets/730ce012-fc9c-4d42-a878-182b3c21fb54)
可以修改这个位置的文件中的  lua_package_path

**Comment by 9a6c** [comment]
好的， 我尝试在waf配置中新增，重载是正常的。
请问这个操作应该配置在哪里？ 我尝试配置在waf中，重载会报错。 我在尝试部署旧的项目，不知道不配置有没有影响。
![image](https://github.com/user-attachments/assets/b02040fd-6ec4-4b58-8f53-b577b1e877be)

**Comment by zhengkunwang223** [comment]
> 好的， 我尝试在waf配置中新增，重载是正常的。 请问这个操作应该配置在哪里？ 我尝试配置在waf中，重载会报错。 我在尝试部署旧的项目，不知道不配置有没有影响。 ![image](https://private-user-images.githubusercontent.com/51264840/352998757-b02040fd-6ec4-4b58-8f53-b577b1e877be.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyNDg2NjEsIm5iZiI6MTcyMjI0ODM2MSwicGF0aCI6Ii81MTI2NDg0MC8zNTI5OTg3NTctYjAyMDQwZmQtNmVjNC00YjU4LThmNTMtYjU3N2IxZTg3N2JlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI5VDEwMTkyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZjYzM0Y2U3NDk3MzJjZjljMjJlNzk0MDZhNGNmMTQ4MzExN2M3NTk1ZGI5MzZjZmY0NTAwMTA1YzkyMTY4ZDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.XYRQYr2TauprjAIB1_o6zxTbWDfKD9j5Hbu8hCszCrA)

这个跟 waf 没关系了 我们没有用到这个
